tests/basic_test.go

```diff
@@ -75,7 +76,7 @@ func TestJobSubmission(t *testing.T) {
 		t.Error(err.Error())
 	}
 
-	err = spark.WaitUntilSucceeded(job)
+	err = spark.WaitUntilSucceeded(10*time.Minute, job)
```

Comment:
should we declare `10*time.Minute` as a constant?

Reply:
Done